dev/com.ibm.ws.jaxws.web/bnd.bnd

```diff
@@ -43,36 +41,35 @@ Service-Component: \
      properties:="service.vendor=IBM"
 
 Export-Package: \
-    com.ibm.ws.jaxws.web;version=${exportVer}
+    com.ibm.ws.jaxws.web
 
 Import-Package: \
-   !com.ibm.ws.jaxws.web, \
    com.ibm.wsspi.adaptable.module.adapters, \
    com.ibm.wsspi.webcontainer.collaborator, \
    com.ibm.ws.container.service.metadata, \
    javax.xml.ws.*;version="[2.2,3)", \
-   org.apache.cxf.*;version="[2.6.2,3)", \
-   com.ibm.ws.jaxws;version="[2.0,2.4)", \
-   com.ibm.ws.jaxws.*;version="[2.0,2.4)", \
+   org.apache.cxf.*;version="[3.2,4)", \
    *
 
 Private-Package:\
-   com.ibm.ws.jaxws.web.*
+   com.ibm.ws.jaxws23.web.*
 
-instrument.classesExcludes: com/ibm/ws/jaxws/web/internal/resources/*.class
+instrument.classesExcludes: com/ibm/ws/jaxws23/web/internal/resources/*.class
```

Comment:
com.ibm.ws.jaxws23.web isn't a thing.  It should stay com.ibm.ws.jaxws.web for both of these locations.